packages/main/src/plugin/kubernetes/kubernetes-port-forward-model.ts

```diff
@@ -0,0 +1,80 @@
+/**********************************************************************
```

Comment:
Some of the models here will be used by the frontend, this file need to be moved to `packages/api/src/`